acme/Watch/main.go

```diff
@@ -234,7 +234,7 @@ func runBackground(id int) {
 		run.Unlock()
 
 		line := data[1:] // chop %
-		cmd := exec.Command("/usr/local/plan9/bin/rc", "-c", string(line))
+		cmd := exec.Command("9", "rc", "-c", string(line))
```

Comment:
Perhaps you could use os.Getenv("PLAN9"), and fallback to /usr/local/plan9 if it's empty, to build the path to rc; so you don't change PATH for commands started by Watch.

(I'm actually wondering why the full path to rc is necessary at all: isn't $PLAN9/bin part of the path? Are there other binaries called rc that would take precedence? Maybe so.)

Reply:
OK I have updated the patch so that it resolves `rc` at startup.